.github/workflows/test_backend_uv.yaml

```diff
@@ -32,3 +33,16 @@ jobs:
 
       - name: Run backend integration tests
         run: AUTO_TEST_RUN=y make test-backend-integration
```

Comment:
Sometimes jobs even fail to start. In those cases, it's useful to collect the logs from Ray itself. Only the `/tmp/ray/session_*/logs/` directory is needed, otherwise we could be fetching the entire Python distro in Ray.